calculator/src/components/Inputs.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+
+function Inputs({ calculateTip }) {
+
+  const [total, setTotal] = useState(0);
+  const [tip, setTip] = useState(0);
+
+  const handleChangeTotal = (e) => {
+    setTotal(Number(e.target.value));
+  }
+
+  const handleChangeTip = (e) => {
+    setTip(Number(e.target.value));
+  }
+
+  const handleClick = () => {
```

Comment:
Awsome organization!. 

I think here you chould have called the calculate tip on the onchage event of the fields instead of the o'clock. This wouy you can have a dynamic a real time calculation!.